src/server/server_family.cc

```diff
@@ -549,6 +550,17 @@ void ServerFamily::Init(util::AcceptServer* acceptor, std::vector<facade::Listen
       LOG(WARNING) << "Invalid snapshot time specifier " << save_time;
     }
   }
+
+  string snapshot_cron_exp = GetFlag(FLAGS_snapshot_cron);
```

Comment:
One of the issues here is that `--save_schedule` might also be defined. I think only one of the two should be allowed but not both. 

Also maybe, I find having two separate flags for snapshotting slightly confusing UX. Is it worth merging them to one flag? That way we could first try parsing it a cron, if that fails try out the other path, if that fails throw an error? 

@romange @chakaz what do you say?

Reply:
Regarding the first comment, you are correct and we decided to go for `ERROR` log and then `exit(1)`.
For the second comment, the original issue requested the original flag (`save_time`) to remain for now for backwards compatibility and will probably be completely removed in the future.
And IMO a more confusing UX is a flag that accepts two different input formats, so I guess we'll wait for more comments :)

Reply:
I think that the agreed upon solution (see https://github.com/dragonflydb/dragonfly/pull/1599#discussion_r1278558195) to `exit()` if both are specified is good enough :)